cabal.project

```diff
@@ -1,3 +1,5 @@
 packages: .
 
 allow-newer: htoml-megaparsec:unordered-containers
+allow-newer: toml-parser:base
+allow-newer: tasty-hspec:base
```

Comment:
I just checked and saw that `tasty-hspec` supports `base-4.13`. Maybe this line could be removed?